src/tools/sssctl/sssctl_cache.c

```diff
@@ -718,6 +763,55 @@ errno_t sssctl_netgroup_show(struct sss_cmdline *cmdline,
         return ret;
     }
 
+    return EOK;
+}
+
+errno_t sssctl_gpo_show(struct sss_cmdline *cmdline,
+                        struct sss_tool_ctx *tool_ctx,
+                        void *pvt)
+{
+    struct sssctl_cache_opts opts = {0};
+    const char *attr;
+    errno_t ret;
+    const char *extended_help =
+        "This command requires the domain name to be given when the "
```

Comment:
Hi,

I think this is currently not correct. In my tests I have always provide the domain name even if there is only a single domain. So either the help message should be changes to something like `... requires the domain name to be given because the same policy name (or GUID) might exists ...` or the code should be changed. I would prefer to just change the help message an always require the domain name to be given to make the usage more consistent.

bye,
Sumit

Reply:
Makes sense, domain is now mandatory.